specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json

```
@@ -1614,6 +1614,208 @@
         }
       }
     },
+    "ManagedHttpRoutePatch": {
+      "description": "A managed http route to update",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Application-specific metadata in the form of key-value pairs."
+        }
+      }
+    },
+    "ManagedHttpRoute": {
+      "description": "Managed certificates used for Custom Domain bindings of Container Apps in a Managed Environment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Certificate resource specific properties",
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "The scale configuration of the session pool.",
+              "$ref": "#/definitions/HttpRouteProvisioningState"
+            },
```

This is also a read-only property, correct?


```suggestion
            "provisioningState": {
              "description": "The scale configuration of the session pool.",
              "$ref": "#/definitions/HttpRouteProvisioningState"
              "readOnly": true
            },
```